middle_end/flambda2/kinds/flambda_kind.mli

```diff
@@ -211,6 +211,11 @@ module With_subkind : sig
 
   val any_value : t
 
+  (* CR lmaurer: Potentially confusing name? *)
```

Comment:
Let's find a better name now, then.  Also could you move this function so it isn't amongst the values just of type "`t`"?

Reply:
I couldn't think of a better name, and I realized that "is this exactly equal to `Flambda_kind.With_subkind.any_value`" is the sort of extremely syntactic thing that only `Flambda_to_fexpr` wants to ask anyway, so I moved it out of here altogether.